### DISCUSSION TITLE
Mistral Nemo download in settings results in server error when trying to use the model

### DISCUSSION THREAD

**Discussion by menelic** [open-webui/open-webui#4141] [General]
# Bug Report
## Description

 When downloading versions of Mistral Nemo listed in the Ollama Model gallery dlwomload works, but trying ti chat with the model will result in  an error message 

## Environment

Ubuntu 22.04
Firefox lVersion 128.0.2 (64-bit)
## Logs and Screenshots

![image](https://github.com/user-attachments/assets/b32a0806-5a92-43a8-b92e-df65e2e5f345)

same error with the Q5 K M model version

**Comment by Collabeta** [comment]
Same. I'm using the bundled container and I'm assuming it's because Mistral-nemo requirements at least Ollama 0.2.8 and the one included is 0.2.1.
